docs/releases/1.9.txt

```diff
@@ -416,6 +419,11 @@ Templates
 * The ``include`` tag now caches parsed templates objects during template
   rendering, speeding up reuse in places such as for loops.
 
+* Support for iterating over
+  :class:`~django.template.backends.django.DjangoTemplates` was added. This
+  includes a ``stream`` method on Template and Node objects, a ``stream`` function on
```

Comment:
Please add links to the new methods and functions. No need to mention `Node` as that's a private API.